### DISCUSSION TITLE
Welcome to blockly-plugins Discussions!

### DISCUSSION THREAD

**Discussion by mark-friedman** [mit-cml/blockly-plugins#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by preetvadaliya** [comment]
Hi @mark-friedman,
I am Preet from India and an open-source contributor and GSOC-21, GSOC-22 student at App Inventor.

Actually, we are building a block programming interface for primary school students, and for that, I am looking for how to create a small flyout on the block that will open on the mouse enter event.

**Reply by mark-friedman** [reply]
Sorry, @preetvadaliya, I just noticed your message.  What would the flyout contain?

**Reply by preetvadaliya** [reply]
Actually, we want to add the getter and setter blocks of the corresponding variable block inside the flyout.
Currently, We solved this using a custom context menu action.

**Reply by mark-friedman** [reply]
> Actually, we want to add the getter and setter blocks of the corresponding variable block inside the flyout.

So this seems just like the lexical variable blocks defined in this repo.  Why not use them?
